### PR TITLE
{math}[foss/2018b] rjags v4-8 w/ R 3.5.1 + JAGS v4.3.0

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2594.golett.os - Linux centos linux 7.5.1804, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/b8c40c4ccd66e6b903793ff13c94958b for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node3117.skitty.os - Linux centos linux 7.5.1804, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/4802224e05dbd1912489fbd9d976f626 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2097.delcatty.os - Linux centos linux 7.5.1804, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/25bb93f4fc9074d85bf49fbaac486bc0 for a full test report.

File: easybuild/easyconfigs/j/JAGS/JAGS-4.3.0-foss-2018b.eb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+description = """JAGS is Just Another Gibbs Sampler.  It is a program for analysis 
+ of Bayesian hierarchical models using Markov Chain Monte Carlo (MCMC) simulation  """
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+
+source_urls = [('http://sourceforge.net/projects/mcmc-jags/files/JAGS/%(version_major)s.x/Source/', 'download')]
+sources = [SOURCE_TAR_GZ]
+checksums = ['8ac5dd57982bfd7d5f0ee384499d62f3e0bb35b5f1660feb368545f1186371fc']
+
+configopts = ' --with-blas="-lopenblas" --with-lapack="-lopenblas" '
```
[the remark this answers is not in the document]
Comment (boegel) [reply easybuild/easyconfigs/j/JAGS/JAGS-4.3.0-foss-2018b.eb]: @akesandgren When building with `intel`, we use:

```
configopts = ' --with-blas="-lmkl" --with-lapack="-lmkl" '
```

which is not the same as using `$LIBBLAS` or `$LIBLAPACK`

[the remark this answers is not in the document]
Comment (akesandgren) [reply easybuild/easyconfigs/j/JAGS/JAGS-4.3.0-foss-2018b.eb]: Yeah, just checking.

**Comment by akesandgren** [comment]
Going in, thanks @boegel!

**Pull request merged**